### DISCUSSION TITLE
Using catkin_make under catkin_ws, have error messages.

### DISCUSSION THREAD

**Discussion by coa06yl** [ifm/ifm3d#375] [Q&A]
Dear all,

When I catkin_make, I got error messages below. It will be great if anyone can help me solve the problem. Many thanks!
![Screenshot from 2022-10-31 17-43-14](https://user-images.githubusercontent.com/7136290/198979009-05372abb-9592-47b1-8cde-a5550c29cde6.png)

**Comment by lola-masson** [comment]
Hey @coa06yl,

At this time ifm3d-ros is only compatible with ifm3d 0.93.0. We are in the process of updating the ros drivers, but at this time you will need to use ifm3d 0.93.0.

Sorry for the inconvenience.

**Reply by coa06yl** [reply]
Many thanks!
